### PR TITLE
[images] Added new image to build the acs_engine_prow_test one

### PR DESCRIPTION
**Pull request by papagalu**

### CODE REVIEW

**Comment by test-e2e-win** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: 
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **papagalu**

Assign the PR to them by writing `/assign @papagalu` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/e2e-win/e2e-win-prow-deployment/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: images/acs_engine_prow_build/build_acs_e2e.sh [hunk 1]
```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function cleanup_binfmt_misc () {
```
Comment (adelina-t) [inline images/acs_engine_prow_build/build_acs_e2e.sh]: why?

**Comment by test-e2e-win** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/e2e-win/e2e-win-prow-deployment/pull/20#issuecomment-406193328" title="Approved">adelina-t</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/e2e-win/e2e-win-prow-deployment/blob/master/OWNERS)~~ [adelina-t]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**